projects/plugins/jetpack/modules/widgets/top-posts.php

```diff
@@ -802,8 +814,8 @@ public function get_posts( $post_ids, $count, $types = array( 'post', 'page' ) )
  */
 function jetpack_do_top_posts_widget( $instance ) {
 	// Post Types can't be entered as an array in the shortcode parameters.
-	if ( isset( $instance['types'] ) && is_string( $instance['types'] ) ) {
-		$instance['types'] = explode( ',', $instance['types'] );
+	if ( isset( $instance['types'] ) && is_array( $instance['types'] ) ) {
+		$instance['types'] = implode( ',', $instance['types'] );
```

Comment:
What's going on here?

Reply:
I reverted unwanted changes. Can be ignored now.